mkosi.images/system/mkosi.conf.d/10-arch/mkosi.build.chroot

```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 set -e
 
+if ((NO_BUILD)); then
```

Comment:
I'd probably change this to `if [[ "${NO_BUILD:-0}" =~ ^(1|yes|true)$ ]]; then`, i.e. what we already do in https://github.com/systemd/systemd/blob/453ce77c072e43cdd51f8a8bf8201595c98f9e64/test/run-integration-tests.sh#L57)

As parsing this as an arithmetic expression covers only the integer values.

Also, there's an extra semicolon after each `exit`.